### PR TITLE
CHA-898: add with_root_parent option

### PR DESCRIPTION
**Pull request by tomglvng**

### CODE REVIEW

File: content/swagger/parameters.yaml [hunk 1]
```diff
@@ -189,3 +189,10 @@ with_enriched_attributes:
   required: false
   type: boolean
   x-from-version: "SaaS"
+with_root_parent:
+  name: with_root_parent
+  in: query
+  description: Retrieve the root parent product model code of a variant
```
Comment (hogren) [inline content/swagger/parameters.yaml:195]: For consistency with the rest of the doc, I think `Return` is better than `Retrieve`.

File: content/swagger/resources/products_uuid/definitions/examples/products_uuid.yaml [hunk 2]
```diff
@@ -26,6 +26,7 @@ x-SaaS: {
         "family": "tshirt",
         "groups": [],
         "parent": null,
+        "root_parent": "the_code_of_the_root_product_model",
```
Comment (hogren) [inline content/swagger/resources/products_uuid/definitions/examples/products_uuid.yaml:29]: I think it's better to find a example more realistic. May be you can change the name to "red top XL" for example and rename the root model to «top»

File: content/swagger/resources/products_uuid/definitions/examples/products_uuid.yaml [hunk 3]
```diff
@@ -26,6 +26,7 @@ x-SaaS: {
         "family": "tshirt",
         "groups": [],
         "parent": null,
```
Comment (hogren) [inline content/swagger/resources/products_uuid/definitions/examples/products_uuid.yaml:28]: Warning, if there is no parent, there is no root parent.

**Review by val-- — APPROVED** [review]

File: content/swagger/resources/products_uuid/definitions/product_uuid.yaml [hunk 4]
```diff
@@ -313,6 +320,7 @@ example: {
       ]
     }
   },
+  "root_parent": "the_code_of_the_root_product_model",
```
Comment (hogren) [inline content/swagger/resources/products_uuid/definitions/product_uuid.yaml:323]: Idem note about naming.

**Review by hogren — APPROVED** [review]

**Comment by akeneo-secops** [comment]
### Gitleaks PR check
🟢 We detected no secrets within this PR! Good job 🎉

**Pull request merged**